README.md

```diff
@@ -1,41 +1,41 @@
-# checkbox-provider-arm
-This is a checkbox provider for all ARM architecture platforms. And it will be built as SNAP named *checkbox-arm*. 
-You can define specific plugs to connect to it and start using the test jobs and plans included in checkbox-provider-arm.
+# checkbox-provider-ce-oem
+This is a checkbox provider for both IoT and PC devices. And it will be built as SNAP named *checkbox-ce-oem*. 
+You can define specific plugs to connect to it and start using the test jobs and plans included in checkbox-provider-ce-oem.
 
 # Getting started
-checkbox-arm will define a slot *provider-arm* to allow checkbox-project to connect to access the test jobs and plans.
+checkbox-ce-oem will define a slot *provider-ce-oem* to allow checkbox interface sanp to connect to access the test jobs and plans.
 
-## In checkbox-project
-You have to modify two parts and rebuild your SNAP of checkbox-project.
+## In checkbox interface snap
+You have to modify two parts and rebuild your SNAP of checkbox interface snap.
 ### snapcraft.yaml
-Add a plug into plugs section in *snapcraft.yaml* of your checkbox-project.
+Add a plug into plugs section in *snapcraft.yaml* of your checkbox interface snap.
 ```
 example:
 
 plugs:
-    provider-arm:
+    provider-ce-oem:
     interface: content
-    target: $SNAP/providers/checkbox-provider-arm
-    default-provider: checkbox-arm
+    target: $SNAP/providers/checkbox-provider-ce-oem
+
 ```
 ### wrapper_local
-Add export PATH for checkbox-arm in *wrapper_local* of your checkbox-project.
+Add export PATH for checkbox-ce-oem in *wrapper_local* of your checkbox interface snap.
 ```
 example:
-export PATH="$PATH:$SNAP/usr/bin:$SNAP/usr/sbin:$SNAP/sbin:/snap/bin:$SNAP/bin:/snap/checkbox-arm/current/usr/bin/:/snap/checkbox-arm/current/usr/sbin"
+export PATH="$PATH:$SNAP/usr/bin:$SNAP/usr/sbin:$SNAP/sbin:/snap/bin:$SNAP/bin:/snap/checkbox-ce-oem/current/usr/bin/:/snap/checkbox-ce-oem/current/usr/sbin"
 ```
-### After rebuild SNAP for checkbox-project
-Install the SNAP of checkbox-project and checkbox-arm. Connect slot and plug of *provider-arm*.
+### After rebuild SNAP for checkbox interface snap
+Install the SNAP of checkbox interface snap and checkbox-ce-oem. Connect slot and plug of *provider-ce-oem*.
 
-`$ sudo snap connect checkbox-project:provider-arm checkbox-arm`
+`$ sudo snap connect checkbox:provider-ce-oem checkbox-ce-oem`
```

Comment:
we should connect to correct slot?
$ sudo snap connect checkbox:provider-ce-oem checkbox-ce-oem:provider-ce-oem

Reply:
This command should be fine.  `sudo snap connect checkbox:provider-ce-oem checkbox-ce-oem`
It will connect the plug "provider-ce-oem" on checkbox interface snap with slot "provider-ce-oem" on checkbox-ce-oem.